### DISCUSSION TITLE
Does repairing windows with an iso file remove Windows10Debloater effects?

### DISCUSSION THREAD

**Discussion by JovenCalixto** [Sophia-Community/SophiApp#197] [Q&A] [answered]
Hello everyone

Recently I started using SophiApp in my Windows 10 22H2 and, when I started it, it told me that my system may be unstable because I used Windows10Debloater by Sycnex before. I know that I can remove this message by removing the corresponding Temp folder, but my question is the following.

Before using SophiApp, as I read that message, I repaired Windows by using Windows iso file and keeping files and programs. Does that remove the doings of Windows10Debloater completely or do I need to do a fresh install of the OS in order to make my system stable.

Thank you

**Comment by farag2** [comment, marked as answer]
Sorry for the late reply. I have no answer for you. :( Who knows? I strongly recommend to re-install from the scartch to minimize the damage made by that trojan. I think this will the right decision.

**Reply by JovenCalixto** [reply]
Ok, thank you so much, I'll reinstall it as I don't have anything important in that PC.

Also, thank you so much for the creation of this script.
